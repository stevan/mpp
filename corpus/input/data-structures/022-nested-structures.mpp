my $data = [1, +{ "x" => [2, 3] }];