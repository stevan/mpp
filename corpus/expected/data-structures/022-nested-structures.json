[
  {
    "type": "Declaration",
    "declarator": "my",
    "variable": {
      "type": "Variable",
      "name": "$data"
    },
    "initializer": {
      "type": "ArrayLiteral",
      "elements": [
        {
          "type": "Number",
          "value": "1"
        },
        {
          "type": "HashLiteral",
          "pairs": [
            {
              "key": {
                "type": "String",
                "value": "\"x\""
              },
              "value": {
                "type": "ArrayLiteral",
                "elements": [
                  {
                    "type": "Number",
                    "value": "2"
                  },
                  {
                    "type": "Number",
                    "value": "3"
                  }
                ]
              }
            }
          ]
        }
      ]
    }
  }
]